### PR TITLE
test: removing unused commands from Commands file

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
RCA: There are many unused commands within commands.js file

Solution:
Checked the commands in both CE and EE repository and we are removing those commands whcih are not used in any tests as part of cleanup activity

/ok-to-test tags="@tag.All"

> [!WARNING]
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/9177375213>
> Commit: 925d75886a1d3db4bd9f065ab13dd88c56808c8a
> Cypress dashboard url: <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=9177375213&attempt=1" target="_blank">Click here!</a>
> It seems like **no tests ran** 😔. We are not able to recognize it, please check workflow <a href="https://github.com/appsmithorg/appsmith/actions/runs/9177375213" target="_blank">here.</a>

### CODE REVIEW

**Review by trishaanand — DISMISSED** [review]

File: app/client/cypress/support/commands.js [hunk 1]
```diff
@@ -2110,3 +1459,21 @@ Cypress.Commands.add(
     cy.get(tooltipSelector).should("be.visible").and("contain", expectedText);
   },
 );
+
+Cypress.Commands.add(
```
Comment (trishaanand) [inline app/client/cypress/support/commands.js:1463]: Why did we spend time re-writing paste command?

Comment (NandanAnantharamu) [reply app/client/cypress/support/commands.js:1463]: i had mistakenly removed this commad i added it back, its not rewritten. I wasnt able to figure out as it was a chained command

**Review by trishaanand — APPROVED** [review]

**Pull request merged**